collect_app/src/main/java/org/odk/collect/android/tasks/MediaLoadingTask.java

```diff
@@ -97,7 +97,7 @@ protected void onPostExecute(File result) {
 
         ODKView odkView = formEntryActivity.get().getCurrentViewIfODKView();
```

Comment:
This null check seems unrelated now that the data is set by the activity. `FormEntryActivity.setBinaryWidgetData` does that check anyway so I think it's clearer to remove it.